charts/wga/values.yaml

```
@@ -134,5 +135,9 @@ clusterClient:
   #   memory: 128Mi
 
 unbound:
+  welcomeImage:
+    name: "wga-welcome"
```

the image is called `welcome-page` in the ctr. (`Failed to pull image "ctr.0x.pt/wga/wga-welcome:main": rpc error: code = NotFound desc = failed to pull and unpack image "ctr.0x.pt/wga/wga-welcome:main": failed to resolve reference "ctr.0x.pt/wga/wga-welcome:main": ctr.0x.pt/wga/wga-welcome:main: not found`)

---

welp that was stupid
```suggestion
    name: "welcome-page"
```